packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -222,14 +236,15 @@ class RealAccountSignup extends React.Component {
                       : null,
                   body: ModalLoginPrompt,
               };
-        const has_close_icon = this.active_modal_index < 2 || this.active_modal_index === 4;
+        const has_close_icon = [0, 1, 5, 6].includes(this.active_modal_index);
```

Comment:
would be easier to read if you replace 0, 1, 5, 6 + all the other mentions of index in the other places with the constants instead? e.g. has_close_icon = [ACCOUNT_WIZARD, ...]